src/rmp/src/logic_cut.cpp

```diff
@@ -331,22 +369,21 @@ utl::UniquePtrWithDeleter<abc::Abc_Ntk_t> LogicCut::BuildMappedAbcNetwork(
   return abc_network;
 }
 
-sta::Instance* GetLogicalParentInstance(
-    const std::unordered_set<sta::Instance*>& cut_instances_,
-    sta::dbNetwork* network,
-    utl::Logger* logger)
+sta::Instance* GetLogicalParentInstance(sta::InstanceSet& cut_instances,
+                                        sta::dbNetwork* network,
+                                        utl::Logger* logger)
 {
   // In physical designs with hierarchy we need to figure out
   // the parent module in which instances should be placed.
   // For now lets just grab the first Instance* we see. RMP
   // shouldn't be doing anything across modules right now, but
   // if that changes I assume things will break.
```

Comment:
How is it constrained to a single module?

Reply:
It won't walk through module boundries in the STA search.